src/FileList/FileExtInfoLoader.cc

```diff
@@ -712,12 +709,12 @@ void FileExtInfoLoader::AddInitialComputedEntries(const std::string& hdu, CARTA:
     }
     AddDataTypeEntry(extended_info, data_type, equivalent_type);
 
-    AddShapeEntries(extended_info, shape, spatial_axes, spectral_axis, stokes_axis, render_axes, depth_axis, axes_names);
+    AddShapeEntries(extended_info, shape, axes, axes_names);
 
     if (compressed_fits) {
         compressed_fits->SetShape(shape);
-        compressed_fits->SetSpectralAxis(depth_axis);
-        compressed_fits->SetStokesAxis(stokes_axis);
+        compressed_fits->SetSpectralAxis(axes.spectral); // TODO: this was set to depth axis previously -- which is correct???
```

Comment:
There is no need to set spectral and stokes axis in CompressedFits, these values are never used.  I commented out the lines here as well as the set/get methods in CompressedFits and it still worked. The shape, however, is used later for extended file info computed entries.  For performance reasons, all file browser info is from the headers only (using cfitsio), rather than opening the image which decompresses the file into memory.

Reply:
So at some point these axes needed to be saved but is now unnecessary.